android-app/app/src/main/java/de/dhbw/ka/se/fibo/models/Item.java

```diff
@@ -1,17 +1,19 @@
 package de.dhbw.ka.se.fibo.models;
 
+import java.math.BigDecimal;
+
 public class Item {
     private float amount;
```

Comment:
why is our amount an float ?

Reply:
This is something we should discuss later on. Could you write it down somewhere so that we wont forget it?